src/ToricVarieties/AlgebraicCycles/special_attributes.jl

```diff
@@ -0,0 +1,24 @@
+########################
+# 1: Special attributes of toric varieties
+########################
+
+@doc Markdown.doc"""
+    chow_ring(v::AbstractNormalToricVariety)
+
+Return the Chow ring of the simplicial and complete toric variety `v`.
+
+# Examples
+```jldoctest
+julia> p2 = projective_space(NormalToricVariety, 2);
+
+julia> ngens(chow_ring(p2))
+3
+```
+"""
+function chow_ring(v::AbstractNormalToricVariety)
+    if ((issimplicial(v) == false) || (iscomplete(v) == false))
```

Comment:
A matter of taste, but I find this much more readable:
```suggestion
    if !issimplicial(v) || !iscomplete(v)
```

Reply:
Thank you! I will adjust this more generally now.